src/v2/view-builder/views/EnrollProfileUpdateView.js

```diff
@@ -51,5 +51,19 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+
+  postRender() {
+    BaseView.prototype.postRender.apply(this, arguments);
+    const uiSchema = this.options.currentViewState.uiSchema;
+    /**
+     * As per requirement of this flow set secondEmail default to empty string
+     * ideally server should have passed default string in remediation
+     */
+    uiSchema.forEach(input => {
+      if (input.name === 'userProfile.secondEmail') {
```

Comment:
What's the value of secondEmail at this point, before setting it to an empty string?

Reply:
Will secondEmail always be sent by the server?

Reply:
So this flow has a special case for widget (first form that has all fields as optional) where backend expect `secondEmail` to be sent as `empty` if user clicked on `finish` button instead of `skip`.. Before setting it to the empty the `secondEmail` doesn't exist and hence the only value sent back to the server is `stateHandle`..

>Will secondEmail always be sent by and to the server?I

If it exists in remediation, then yes.

Reply:
Maybe an obvious question but just want to be sure, is there a scenario where the server might set a value for this and we override it to an empty string here?

Reply:
@kamaldeepcheema-okta that won't happen, since this is an enrollment flow which means prompt for enrollment if not already enrolled.. Enrolled user will not see `secondEmail` field..

Reply:
I thought so, thanks for confirming!